tests/WE2E/run_WE2E_tests.py

```diff
@@ -463,44 +490,55 @@ def setup_logging(logfile: str = "log.run_WE2E_tests", debug: bool = False) -> N
-                        help='Explicitly set USE_CRON_TO_RELAUNCH for all experiments; this '\
-                             'option disables the "monitor" script functionality')
-    parser.add_argument('--cron_relaunch_intvl_mnts', type=int,
-                        help='Overrides CRON_RELAUNCH_INTVL_MNTS for all experiments')
-    parser.add_argument('--opsroot', type=str,
-                        help='If test is for NCO mode, sets OPSROOT (see config_defaults.yaml for '\
-                             'more details on this variable)')
-    parser.add_argument('--print_test_info', action='store_true',
-                        help='Create a "WE2E_test_info.txt" file summarizing each test prior to'\
-                             'starting experiment')
-    parser.add_argument('--debug_tests', action='store_true',
-                        help='Explicitly set DEBUG=TRUE for all experiments')
-    parser.add_argument('--verbose_tests', action='store_true',
-                        help='Explicitly set VERBOSE=TRUE for all experiments')
-
-    parser._action_groups.append(optional)
-
-    args = parser.parse_args()
+    ap.add_argument('-c', '--compiler', type=str,
+                    help='Compiler used for building the app', default='intel')
+    ap.add_argument('-d', '--debug', action='store_true',
+                    help='Script will be run in debug mode with more verbose output')
+    ap.add_argument('-q', '--quiet', action='store_true',
+                    help='Suppress console output from workflow generation; this will help '\
+                         'keep the screen uncluttered')
+    ap.add_argument('-p', '--procs', type=int,
+                    help='Run resource-heavy tasks (such as calls to rocotorun) in parallel, '\
+                         'with provided number of parallel tasks', default=1)
+    ap.add_argument('-l', '--launch', type=str, choices=['python', 'cron', 'none'],
+                    help='Method for launching jobs. Valid values are:\n'\
+                         ' python: [default] Monitor and launch experiments using monitor_jobs.py\n'
+                         ' cron:   Launch expts using ush/launch_FV3LAM_wflow.sh from crontab\n'\
+                         ' none:   Do not launch experiments; only create experiment directories',
+                         default="python")
+
+
+    ap.add_argument('--modulefile', type=str, help='Modulefile used for building the app')
+    ap.add_argument('--run_envir', type=str,
+                    help='Overrides RUN_ENVIR variable to a new value ("nco" or "community") '\
+                         'for all experiments', default='')
+    ap.add_argument('--expt_basedir', type=str,
+                    help='Explicitly set EXPT_BASEDIR for all experiments')
+    ap.add_argument('--exec_subdir', type=str,
+                    help='Explicitly set EXEC_SUBDIR for all experiments')
+    ap.add_argument('--use_cron_to_relaunch', action='store_true',
```

Comment:
@mkavulich Seeing the cron option reminded me of a bug that I've run into with the `run_WE2E_tests.py` script, which is that when the user's cron table contains a commented line (job) that is exactly identical to the line that this script would otherwise insert into the table, the script doesn't insert the line (so the experiment doesn't get (re)launched).  I guess it's because the script doesn't check that the existing line is commented out and so thinks the line is already in the table.  Can you run a quick test to see if that is still happening?  Thanks.

Reply:
@mkavulich Just realized the bug (if it still exists) might be in the script `get_crontab_contents.py`.

Reply:
@gsketefian Just included a fix for that bug as well, feel free to test it out.

Reply:
@mkavulich Ok, thanks.  I'll trust you that it works!